src/locales/en/en.json

```diff
@@ -29,30 +29,30 @@
 		},
 		"createRepoForm":{
 			"heading" : "Create Repository",
-			"wouldYouLikeToCreate": "This repository does not yet exist, would you like to create it?",
+			"wouldYouLikeToCreate": "This repository does not exist yet, would you like to create it?",
 			"repo": {
 				"label": "Repository"
 			},
 			"description": {
 				"label":  "Description",
 				"placeholder": "A short description of your repository.",
-				"defaultValue": "Created by CWRC",
-				"helpText": "The description will appear in the GitHub page for your new repository."
+				"defaultValue": "Created with CWRC-Writer",
+				"helpText": "The description will appear in the readme file of your new repository."
```

Comment:
Repo Description is metadata. It shows up on the top of the GitHub page, near the title. On CWRC-Writer, it appears next to the repo's title when navigating on the load dialog. So, it has nothing to do with the readme file. Can you please change back to the original form?

Reply:
done with the latest commit